sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -136,6 +136,7 @@ def main(unused_argv):
     service_descriptor = endpoints_pb2.ApiServiceDescriptor()
     text_format.Merge(os.environ['CONTROL_API_SERVICE_DESCRIPTOR'],
                       service_descriptor)
+    _load_avro_coder(sdk_pipeline_options)
```

Comment:
Can you explain why we need to import the AvroCoder here (but not the other coders). Can we load coders in a uniform way ?

Reply:
Thanks for pointing this out. Will fix this (this is from old design when I thought that Avro coder is only for testing xlang).